src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -215,7 +215,12 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
     ucs_log_level_t    log_lvl;
     ucs_status_t       status;
 
-    ucs_assert(ep != NULL);
+    if (ep == NULL) { /* EP is already destroyed */
+        ucs_diag("Can't find EP for iface %p, qp_num %u, EP already destroyed",
+                 iface, qp_num);
```

Comment:
How can this happen, ep_flush should have waited until all completions arrived
@Artemy-Mellanox WDYT?